classes/output/question_attempt_renderer.php

```diff
@@ -84,10 +84,19 @@ public function render_review_next_button(capquiz_question_attempt $attempt) {
     private function render_progress(capquiz_user $user) {
         $questionlist = $this->capquiz->question_list();
         $percent = $questionlist->next_level_percent($user->rating());
+	$abspercent = $percent ;
+        if ( $percent >= 0 ) {
+           $progressclass = "capquiz-quiz-progress-fill" ;
+        } else {
+           $progressclass = "capquiz-quiz-progress-unfill" ;
```

Comment:
This seems a little too complex than it needs to be. Is this rendered from other places? If not, it's best to let mustache handle display/rendering (style and layout), and keep PHP to prepare and propagate data.